README.md

```diff
@@ -81,10 +86,56 @@ tunnelx -auth <your_api_key>
 
 To keep tunnelx running continuously in the background, follow these instructions based on your operating system:
 
-```sh
 # Linux & MacOS
+
 nohup tunnelx -auth <your_api_key>
 
 # Windows
+
 start /B tunnelx -auth <your_api_key>
+
+## Running as a Systemd Service (Linux)
+
+For a more robust solution on Linux systems, you can run tunnelx as a systemd service. This ensures the service automatically starts on boot and handles terminal disconnections gracefully.
+
+### 📖 How to Use
+
+1. **Save the installation script:**
+
+   ```bash
+   nano install_tunnelx_service.sh
+   ```
+
+   (Copy the content from the `install_tunnelx_service.sh` file present in the repository root and paste it into this local file, then save and exit)
+
+2. **Make it executable:**
+
+   ```bash
+   chmod +x install_tunnelx_service.sh
+   ```
+
+3. **Run the installation script:**
+
+   ```bash
+   ./install_tunnelx_service.sh
+   ```
+
+   The script will prompt you to enter your PDCP API key during installation.
+
+### 📋 Monitoring and Logging
```

Comment:
we can directly give single command to install something like 
https://stackoverflow.com/a/49063127